package.json

```diff
@@ -27,7 +27,7 @@
     "url": "git@github.com:sshev/karma-remap-coverage.git"
   },
   "dependencies": {
-    "remap-istanbul": "0.8.4"
+    "remap-istanbul": "0.10.1"
```

Comment:
Worth putting a `^` so we get any patch fixes of `0.10.*`?

Reply:
@paulblyth Sounds good to me, I'll update and rebase.